modules/ocf/manifests/packages/firefox.pp

```diff
@@ -0,0 +1,19 @@
+class ocf::packages::firefox {
+  package {
+    'firefox':
+      ensure => present;
+    'iceweasel':
+      ensure => absent;
+  }
+
+  file {
+    # disable caching, history, blacklisting, and set homepage
+    '/etc/firefox/profile/prefs.js':
```

Comment:
Well, for one, [localstore.rdf seems to be deprecated](https://support.mozilla.org/en-US/kb/firefox-support-troubleshooting-guide#w_reset-xulstore-or-localstore-file). I suppose we should move be using xulstore.json instead.

Most likely, we can just put one of the two in `/etc/firefox`, which is where prefs.js goes now. Test it on a lab computer.